scripts/process_data_dcm-zurich.sh

```diff
@@ -81,6 +79,50 @@ label_if_does_not_exist(){
   sct_qc -i ${file}.nii.gz -s ${file_seg}_labeled_discs.nii.gz -p sct_label_utils -qc ${PATH_QC} -qc-subject ${SUBJECT}
 }
 
+# Check if manual canal segmentation file already exists. If it does, copy it locally.
+# If it doesn't, perform automatic canal segmentation
+segment_canal_if_does_not_exist() {
+  local file="$1"
+  local contrast="$2"
+  # Update global variable with segmentation file name 
+  # If the segmentation of the canal already exist, would it have this name?
+  FILESEG="${file}_label-canal_mask"
+  FILESEGMANUAL="${PATH_DATA}/derivatives/labels/${SUBJECT}/anat/${FILESEG}-manual.nii.gz"
```

Comment:
the name of the canal segmentation should be `*_label-canal_seg`, and without the addition of the manual
```suggestion
  FILESEG="${file}_label-canal_seg"
  FILESEGMANUAL="${PATH_DATA}/derivatives/labels/${SUBJECT}/anat/${FILESEG}.nii.gz"
```

See  intranet [here](https://intranet.neuro.polymtl.ca/data/dataset-curation.html#derivative-entities) for ref